Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LayerIntegrationTest.kt

```diff
@@ -596,8 +596,8 @@ class LayerIntegrationTest {
         LayerMenuViewInteraction.onLayerMenuView()
             .performOpen()
 
```

Comment:
Visibility is spelled wrong in the method name. :/

Reply:
corrected